server/chat-commands.js

```diff
@@ -631,7 +631,7 @@ const commands = {
 	unaway: 'back',
 	unafk: 'back',
 	back(target, room, user) {
-		if (user.isAway) user.setBack(); // Placebo but covers weird bugs
+		user.setBack(); // Placebo but covers weird bugs
```

Comment:
No longer a placebo?

Reply:
no still a placebo because sending ANY command works still. Just not /cmd

Reply:
I don't think thats the behavior I'd want - I think unless the command I write is broadcast in chat I'd like to keep my explicit `/away` status. I'd like to even be able to send PMs while away (I'm basically asking for MSN's "Appear Offline"). Unless I make a public action I think I should be able to stay `/away`?

Reply:
I think PMs should definitely mark you as back. As for other commands, maybe, idk. I think this is just an ok starting point we can always add more specialized behavior.

Reply:
Definitely should no longer be a placebo at this point?